modules/cert-manager-install-console.adoc

```diff
@@ -24,12 +24,12 @@ You can use the web console to install the {cert-manager-operator}.
 . Select the *{cert-manager-operator}* and click *Install*.
 
 . On the *Install Operator* page:
-.. The *Update channel* is set to *tech-preview*, which installs the latest Technology Preview release of the {cert-manager-operator}.
-.. The *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the Operator-recommended `openshift-cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
-.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `openshift-cert-manager-operator`.
+.. The *Update channel* defaults to *stable-v1*, which installs the latest stable release of the {cert-manager-operator}.
+.. The *Installation Mode* is set to *A specific namespace on the cluster*.
```

Comment:
```suggestion
.. Ensure that the Installation Mode is set to *A specific namespace on the cluster*.
```

Reply:
Removed this sentence all together since there is no action needed on the users part here anymore